R/utils-co2_upper_lower.R

```diff
@@ -2,11 +2,24 @@ create_co2_range <- function(data) {
   data |>
     summarize_range(.data[[grep("co2_footprint", names(data), value = TRUE)]],
                     .by = c("grouped_by", "risk_category")) |>
-    jitter_range() |>
+    jitter_range(amount = 0.9) |>
+    stop_if_percent_noise_more_than_100() |>
```

Comment:
It would be useful to pass `amount` as an argument, rather than hard-coding it. 


``` r
devtools::load_all(".")
#> ℹ Loading tiltIndicatorAfter

set.seed(1)


# As I user I want to be able to change the amount of jitter
jitter_range(tibble(min = 1:5, max = 1:5), amount = 1)
#> # A tibble: 5 × 4
#>     min   max min_jitter max_jitter
#>   <int> <int>      <dbl>      <dbl>
#> 1     1     1      0.531       1.80
#> 2     2     2      1.49        3.78
#> 3     3     3      2.56        3.96
#> 4     4     4      0.734       5.03
#> 5     5     5      2.02        9.38
jitter_range(tibble(min = 1:5, max = 1:5), amount = 10)
#> # A tibble: 5 × 4
#>     min   max min_jitter max_jitter
#>   <int> <int>      <dbl>      <dbl>
#> 1     1     1      -2.88       1.05
#> 2     2     2      -2.94      10.7 
#> 3     3     3      -8.22      32.5 
#> 4     4     4      -5.27      13.6 
#> 5     5     5     -22.0       32.7
jitter_range(tibble(min = 1:5, max = 1:5), amount = 100)
#> # A tibble: 5 × 4
#>     min   max min_jitter max_jitter
#>   <int> <int>      <dbl>      <dbl>
#> 1     1     1      -85.9       21.8
#> 2     2     2     -105.       189. 
#> 3     3     3      -88.0       55.6
#> 4     4     4     -264.       300. 
#> 5     5     5     -178.       115.

# But create_co2_range() lacks the `amount` argument
data <- tibble::tribble(
  ~input_co2_footprint,                    ~grouped_by, ~risk_category,
                   100,                          "all",         "high",
                   100,            "input_isic_4digit",         "high",
                   100,            "input_tilt_sector",         "high",
                   100,                   "input_unit",         "high",
                   100, "input_unit_input_isic_4digit",         "high",
                   100, "input_unit_input_tilt_sector",         "high"
)
# Fails
try(create_co2_range(data, amount = 0.5))
#> Error in create_co2_range(data, amount = 0.5) : 
#>   unused argument (amount = 0.5)
```

<sup>Created on 2024-02-26 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

Reply:
Resolved this in next commit. Please check!